src/jit/block.h

```diff
@@ -438,6 +438,8 @@ struct BasicBlock : private LIR::Range
 #define BBF_CLONED_FINALLY_BEGIN 0x100000000 // First block of a cloned finally region
 #define BBF_CLONED_FINALLY_END 0x200000000   // Last block of a cloned finally region
 
+#define BBF_DOMINATED_BY_NOT_NORMAL_ENTRY 0x400000000 // Full set of dominators contains not normal entry.
+
```

Comment:
Fixed, it is a bit unclear because we include call_finally block for ARM and I don't think that we could call it exceptional. I do not know should we mark all finally blocks or not with this flag.
Do we expect better performance if we SetVolatileHint on local variables that live in finally blocks?